sstables/mx/reader.cc

```diff
@@ -1417,7 +1485,14 @@ class mx_sstable_mutation_reader : public mp_row_consumer_reader_mx {
 
         if (_single_partition_read) {
             _read_enabled = (begin != *end);
-            _context = data_consume_single_partition<DataConsumeRowsContext>(*_schema, _sst, _consumer, { begin, *end });
+            if (_slice.options.contains(query::partition_slice::option::reversed)) {
```

Comment:
This should have been saved in a `_reversed` member, to make it easier on the eyes.